OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Lint Fix Weekly Summaries

# Description
Lint Fix `src/component/WeeklySummary/**`

## Main changes explained:
- Change the code style according to ESLint and Prettier

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any user
4. go to dashboard→ summary
5. check the app works as normal
6. set up ESLint and Prettier in VSCode
7. check all files in `src/component/WeeklySummary/**` doesn't have errors

## Screenshots or videos of changes:
![Screenshot 2023-08-29 at 16 09 44 (2)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/700db70f-9980-48c0-a56d-d086663d51b4)
